src/index.html

```diff
@@ -12,7 +12,39 @@
       href="styles/index.scss"
     />
```

Comment:
The href attribute in the link tag should point to a CSS file, not a SCSS file. SCSS files need to be compiled to CSS before they can be used in an HTML file. Please ensure that you link to the compiled CSS file instead.